internal/pkg/cli/pipeline_init.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store/ssm"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/prompt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/spf13/afero"
+)
+
+
+// InitPipelineOpts holds the configuration needed to create a new pipeilne
+type InitPipelineOpts struct {
+	// Fields with matching flags.
+	GitHubRepo  string
+	GitHubAccessToken string
+	AppName string
```

Comment:
Do we need this flag?

I believe we want to pick up all the applications in the workspace.

Reply:
Hm, really? I guess that makes sense -- maybe I'm still confused about how we're making this explicit based on the last IRL discussion.